pkg/dashboard/apiserver/event/fx.go

```diff
@@ -0,0 +1,31 @@
+// Copyright Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package event
+
+import (
+	"github.com/go-logr/logr"
+
+	config "github.com/chaos-mesh/chaos-mesh/pkg/config/dashboard"
+	"github.com/chaos-mesh/chaos-mesh/pkg/dashboard/core"
+)
+
+func Bootstrap(event core.EventStore,
+	workflowStore core.WorkflowStore,
+	conf *config.ChaosDashboardConfig,
+	logger logr.Logger,
+) *Service {
+	return NewService(event, workflowStore, conf, logger.WithName("events"))
+}
```

Comment:
It seems the `Bootstrap` is exactly the same as the `NewService`. I think we do not require to use a new `Bootstrap` method.

Reply:
Thanks, I'll apply.